src/System.IO.MemoryMappedFiles/src/Interop/Interop.Windows.cs

```diff
@@ -211,6 +210,12 @@ internal static partial class mincore
             ref SECURITY_ATTRIBUTES lpFileMappingAttributes, int flProtect,
             int dwMaximumSizeHigh, int dwMaximumSizeLow, string lpName);
 
+        [DllImport(MEMORYDLL, EntryPoint = "CreateFileMappingW", CharSet = CharSet.Unicode, SetLastError = true)]
+        [SecurityCritical]
+        internal static extern SafeMemoryMappedFileHandle CreateFileMapping(IntPtr hFile,
```

Comment:
Sorry.  I was not looking at the complete file (just what GitHub showed in the diff) and didn't realize this was an overload of the SafeHandle version.  What you have is fine.